app/style/list/conversation-list-cell.less

```diff
@@ -59,7 +59,7 @@
 /* elements */
 .conversation-list-cell-availability {
   .availability-state-icon {
-    margin-right: 8px;
+    margin-right: 1px 8px 1px 0;
```

Comment:
This is not possible it should be `margin: 1px 8px 1px 0;`

Reply:
of course. 
thanks!